app/src/main/java/com/nervousfish/nervousfish/activities/BluetoothConnectionActivity.java

```diff
@@ -50,7 +50,7 @@
     private BluetoothAdapter bluetoothAdapter;
     private IBluetoothHandler bluetoothHandler;
     private Set<BluetoothDevice> pairedDevices;
-    private boolean isMaster = false;
+    private boolean isMaster;
```

Comment:
I'm not sure if enforcing code style through an IDE is good practice either, no one is forced to use IntelliJ/Android studio... Preferably we could enforce this through Checkstyle or some kind of linter 🤔